### PR TITLE
Add support for ES6 in uglify task

### PR DESCRIPTION
**Pull request by thornbill**
Adds `uglify-es` for ES6 support in the gulp build

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,7 +23,8 @@
     "gulp-if": "^2.0.2",
     "gulp-uglify": "^3.0.2",
```
Comment (JustAMan) [inline package.json]: why is this still here?

File: gulpfile.js [hunk 2]
```diff
@@ -3,7 +3,9 @@ var gulpif = require('gulp-if');
 var cleanCSS = require('gulp-clean-css');
 var del = require('del');
 var dom = require('gulp-dom');
-var uglify = require('gulp-uglify');
+var uglifyes = require('uglify-es');
+var composer = require('gulp-uglify/composer');
+var uglify = composer(uglifyes, console);
```
Comment (JustAMan) [inline gulpfile.js]: what this compositing does?

Comment (thornbill) [reply gulpfile.js]: This is the method provided by `gulp-uglify` to replace the version of `uglifyjs` that it uses internally. For reference: https://github.com/terinjokes/gulp-uglify#using-a-different-uglifyjs

File: package.json (hunk 1 above)
Comment (thornbill) [reply package.json]: `gulp-uglify` is still being used, but is configured to use `uglify-es` internally.

**Pull request merged**